geoips/plugins/modules/output_checkers/text.py

```diff
@@ -23,6 +23,61 @@
 name = "text"
 
 
+def clear_text(match_path, close_path, bad_path):
+    """Clear output text files so they can be written again."""
+    open(match_path, "w").close()
+    open(close_path, "w").close()
+    open(bad_path, "w").close()
+
+
+def get_test_files():
+    """Return a series of varied text files."""
+    import numpy as np
+    from shutil import copy
+    from os import makedirs
+    from os.path import exists
+    from importlib.resources import files
+
+    savedir = str(files("geoips") / "../../test_data/test_text/pytest") + "/"
```

Comment:
@jsolbrig This is where Evan updated to not use $GEOIPS_PACKAGES_DIR, though perhaps it would be better to just use $GEOIPS_OUTDIRS because if geoips is NOT installed in $GEOIPS_PACKAGES_DIR, ../../ may not be where you expect.

Reply:
Decided to pass in output_path.  See latest file diffs.